ssh-setup.yml

```diff
@@ -0,0 +1,29 @@
+---
+- name: Get IP Address of Atomic Host
+  hosts: localhost
+  become: yes
+  become_method: sudo
+
+  tasks:
+  - name: Wait for 10s to Get the IP
+    pause:
+      seconds: 10
+
+  - name: Get the IP
+    shell: virsh domifaddr atomic-node | tail -2| awk '{print $4}' | cut -d/ -f1
+    register: vm_ip
+
+  - debug: msg="{{ vm_ip.stdout }}"
+
+  - name: ssh-copy-id to the Atomic host
+    shell: ssh-copy-id -i ~/.ssh/id_rsa.pub atomic-user@$"{{ vm_ip.stdout }}"
+
+  - name: Create Inventory file
+    file:
+      path: ./inventory
+      state: touch
+
+  - name: Write Atomic host in Inventory file
```

Comment:
this step can be become obsolete, as you will not have to rerun Ansible when everything is in one playbook.